timesketch/lib/analyzers/yetiindicators.py

```diff
@@ -0,0 +1,129 @@
+"""Index analyzer plugin for Yeti indicators."""
+from __future__ import unicode_literals
+
+from flask import current_app
+import requests
+
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+from timesketch.lib import emojis
+
+
+def build_query_for_indicators(indicators):
+    """Builds an ElasticSearch query for Yeti indicator patterns.
+
+    Prepends and appends .* to the regex to be able to search within a field.
+
+    Returns:
+      The resulting ES query string.
+    """
+    query = []
+    for domain in indicators:
+        query.append('domain:/.*{0:s}.*/'.format(domain['pattern']))
+    return ' OR '.join(query)
+
+
+class YetiIndicators(interface.BaseSketchAnalyzer):
+    """Index analyzer for Yeti threat intel indicators."""
+
+    NAME = 'yetiindicators'
```

Comment:
Do you plan on having more Yeti analyzers? If you only plan on updating this one I opt for renaming this (filename and class and NAME) to simply "yeti".

Reply:
ack